README.md

```diff
@@ -1,73 +1,84 @@
 
-He has endeavored to bring on the inhabitants of our frontiers the merciless Indian savages, whose known rule of warfare is an undistinguished destruction of all ages, sexes, & conditions of existence.
+He has excited domestic insurrections amongst us, and has endeavoured to bring on the inhabitants of our frontiers, the merciless Indian Savages, whose known rule of warfare is an undistinguished destruction of all ages, sexes and conditions.
 
-He has incited treasonable insurrections of our fellow-citizens, with the allurements of forfeiture & confiscation of our property.
+In every stage of these Oppressions We have Petitioned for Redress in the most humble terms: Our repeated Petitions have been answered only by repeated injury. A Prince, whose character is thus marked by every act which may define a Tyrant, is unfit to be the ruler of a free people.
 
-He has waged cruel war against human nature itself, violating it’s most sacred rights of life and liberty in the persons of a distant people who never offended him, captivating & carrying them into slavery in another hemisphere, or to incur miserable death in their transportation thither. This piratical warfare, the opprobium of INFIDEL Powers, is the warfare of the CHRISTIAN king of Great Britain. Determined to keep open a market where MEN should be bought & sold, he has prostituted his negative for suppressing every legislative attempt to prohibit or to restrain this execrable commerce. And that this assemblage of horrors might want no fact of distinguished die, he is now exciting those very people to rise in arms among us, and to purchase that liberty of which he has deprived them, by murdering the people on whom he also obtruded them: thus paying off former crimes committed against the LIBERTIES of one people, with crimes which he urges them to commit against the LIVES of another.
+He has waged cruel war against human nature itself, violating its most sacred rights of life and liberty in the persons of a distant people who never offended him, captivating & carrying them into slavery in another hemisphere, or to incur miserable death in their transportation thither.
+This piratical warfare, the opprobium of INFIDEL Powers, is the warfare of the CHRISTIAN king of Great Britain.
+Determined to keep open a market where MEN should be bought & sold, he has prostituted his negative for suppressing every legislative attempt to prohibit or to restrain this execrable commerce. And that this assemblage of horrors might want no fact of distinguished die, he is now exciting those very people to rise in arms among us, and to purchase that liberty of which he has deprived them, by murdering the people on whom he also obtruded them: thus paying off former crimes committed against the LIBERTIES of one people, with crimes which he urges them to commit against the LIVES of another.
```

Comment:
This is all true, passionate and well said. But could cut both ways since they found a market on our shores and the southern delegate has already expressed their intent to reopen the trade. Better to leave this part out.